app/operations/assignments/create.rb

```diff
@@ -1,7 +1,7 @@
 module Assignments
   class Create < Operation
-    integer :base_project_id, default: Rails.application.secrets["zooniverse_project_id"]
```

Comment:
Thanks @marten ! 593 and 338 for WildCam Gorongosa production, I imagine - I'll need to find out what's needed to be done for staging. (I don't believe there's any WildCam G stuff set up there ☹️  )